### ISSUE TITLE
Setup is suddenly not working anymore

### ISSUE THREAD

**Issue by ilucha101** [psieg/Lightpack#425]
Is there anyone who has problems with their Harware/Software?
My setup:
- Windows 10 20H2
- Arduino Nano with CH340 (newest driver from wch_dot_cn website)
- WS2812B with 60 LEDs
- Prismatik version 5.11.2.25

After upgrading to 20H2, my LEDs are simply not turning on anymore (although I'm not 100% sure if the error happend after the update or shortly after). I already checked if the LEDs get power and basically the setup never changed. I even have a spare fake Arduino Nano and even this one does not work?
In Device-Manager the Arduino is shown as a USB-Serial device with COM3 Port and that is what is configured in Prismatik.
Prismatik though shows no errors with my device, it is simply not turning on. There are no logs to attach here.

I'm simply out of ideas to try/test anymore. Can someone point me in the right direction?

**Comment by ilucha101** [comment]
Is there no one with similar issues or has any advice what to look for?

**Comment by ilucha101** [comment]
Today I bought an Nano Every and tried it again. First of all the Nano every has currently no support in the FastLED library so you have to use a workaround I found here by ngyl88. After that my LED strip worked like before!
So I uploaded the sktech to my previous fake Nano and nothing happens. Seems like my fake Nano is dead (maybe only the according pin). I replaced the fake Nano with another I have lying around and it works flawlessly again!

TL;DR
Fake Nano (partly) died and so did the whole setup. Replaced the fake Nano with another one and everything went back to normal

**Comment by Joeri87** [comment]
> workaround I found here by ngyl88

What workaround do you mean? And does this also count for the NeoPixel?
